plugins/auth-node/src/sign-in/commonSignInResolvers.ts

```diff
@@ -16,6 +16,11 @@
 
 import { createSignInResolverFactory } from './createSignInResolverFactory';
 
+// This splits an email "joe+work@acme.com" into ["joe", "+work", "@acme.com"]
+// so that we can remove the plus addressing. May output a shorter array:
+// ["joe", "@acme.com"], if no plus addressing was found.
+const reEmail = /([^@+]+)(\+[^@]+)?(@.*)/;
```

Comment:
Prolly worth adding `^`...`$` for safety? Matching something like `+a+b@c` gets a bit strange otherwise

Reply:
That's a good idea for sure, have updated the PR